qubes-trust-daemon.cpp

```diff
@@ -141,17 +140,14 @@ void mark_files_as_untrusted(const std::set<std::string> file_paths) {
                 execv("/usr/bin/qvm-file-trust", (char**) qvm_argv);
 
                 // Unreachable if no error
-                perror("execl qvm-file-trust");
-                exit(1);
+                perror("execl qvm-file-trust failed");
```

Comment:
Actually, here exit(1) was ok. Now it will fall through to case -1, then to default...

Reply:
remember this case is in the child process.

Reply:
Whoops, that would not be good in the child process. Too much Golang lately where there's no fall-through in switch statements.

Reverted changed in 93ebb4c